pythonforandroid/recipes/python3/__init__.py

```diff
@@ -21,9 +21,10 @@ class Python3Recipe(GuestPythonRecipe):
     url = 'https://www.python.org/ftp/python/{version}/Python-{version}.tgz'
     name = 'python3'
 
-    patches = ["patches/fix-ctypes-util-find-library.patch"]
+    patches = ['patches/fix-ctypes-util-find-library.patch',
+               'patches/fix-zlib-version.patch']
 
-    depends = ['hostpython3', 'sqlite3', 'openssl', 'libffi']
+    depends = ['hostpython3', 'sqlite3', 'openssl', 'libffi', 'zlib']
```

Comment:
We also could read the zlib version of the android's ndk and make it work without introducing a new recipe, but I think that it's better with a recipe because we can use the last version of zlib (we are using 1.2.11 and android ndk r19b uses 1.2.7)

**Note:** the same applies to python2 recipe

Reply:
Is it important to have zlib fully up to date? I would tend to prefer using the system library.

Reply:
Yeah sticking with simpler & system zlib also sounds smarter to me. Apart from security holes (which I would hope Google would update) I don't see a good reason to deviate

Reply:
Ok, then **let's do it with android's zlib**, I already have the code and it also works, I will push the changes later :wink:

¡¡¡Thanks @inclement and @JonasT!!!